### DISCUSSION TITLE
JustMyCode supports Python and debug versions

### DISCUSSION THREAD

**Discussion by sunyonghua** [microsoft/debugpy#1557] [Q&A]
JustMyCode supports Python and debug versions?

Is Python 3.6.8 and debugpy1.5.1 supported?

**Comment by int19h** [comment]
JMC has been a debugpy feature for a very long time and was supported in 1.5.1.

That said, in general, we don't provide support for older debugpy versions except where they shipped as part of another Microsoft product that is still supported (e.g. VS), and then only in the context of said product. Otherwise if you run into an issue the recommendation is to update to a newer version of debugpy and see if it still repros; although this would mean no Python 3.6 support.
